app/views/layouts/application.html.erb

```diff
@@ -12,14 +12,16 @@
 
 <script>
   $(function() {
+    var ideas = $('.ideas');
+    var search_bar = $('input[name=search]');
     $('form#createIdea').submit(Ideas.create);
```

Comment:
Probably not so much `ideas` but the `search_bar` makes it a little easier to read - Only concerns are that you're using snake case rather than camel case and you may want to name jQuery variables starting like `$searchBar` just to be more idiomatic